src/NLog.StructuredLogging.Json.Tests/nlog.config

```diff
@@ -3,25 +3,15 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       throwExceptions="true"
     autoReload="true">
-
-  <variable name="log_dir" value="${basedir}/../logs"/>
-  <variable name="archive_dir" value="${basedir}/../logs_archive"/>
   <variable name="human_readable_layout" value="${longdate}|${level}|${logger}|${message}|${onexception:EXCEPTION OCCURRED\:${exception:format=type,message,method,stackTrace:maxInnerExceptionLevel=5:innerFormat=shortType,message,method,stackTrace}}"/>
-
   <targets async="true">
     <target xsi:type="Null" name="drop"/>
-
-    <target xsi:type="Debugger" name="debugger" layout="${human_readable_layout}"/>
-
-    <target
-      xsi:type="Console"
-      name="console"
-      layout="${human_readable_layout}"/>
+    <target xsi:type="Debugger" name="Debugger" layout="${human_readable_layout}"/>
+    <target xsi:type="Console" name="Console" layout="${human_readable_layout}"/>
   </targets>
-
   <rules>
-    <logger name="*" minlevel="Trace" writeTo="console"/>
-    <logger name="*" minlevel="Trace" writeTo="debugger"/>
+    <logger name="*" minlevel="Trace" writeTo="Console"/>
+    <logger name="*" minlevel="Trace" writeTo="Debugger"/>
```

Comment:
I was trying to get it working on Travis and linux. it didn't have any effect, so I should probably revert that part